ush/forecast_postdet.sh

```diff
@@ -2,6 +2,9 @@
 
 # Disable variable not used warnings
 # shellcheck disable=SC2034
+
+ntiles=${ntiles:-6}
```

Comment:
is this needed here? there should be several instances where the tiles are looped through in this file aren't there?